src/transaction/log_page_buffer.c

```diff
@@ -370,9 +370,13 @@ static void logpb_fatal_error_internal (THREAD_ENTRY * thread_p, bool log_exit,
 static int logpb_copy_log_header (THREAD_ENTRY * thread_p, LOG_HEADER * to_hdr, const LOG_HEADER * from_hdr);
 STATIC_INLINE LOG_BUFFER *logpb_get_log_buffer (LOG_PAGE * log_pg) __attribute__ ((ALWAYS_INLINE));
 STATIC_INLINE int logpb_get_log_buffer_index (LOG_PAGEID log_pageid) __attribute__ ((ALWAYS_INLINE));
-static int logpb_fetch_header_from_active_log (THREAD_ENTRY * thread_p, const char *db_fullname,
-					       const char *logpath, const char *prefix_logname, LOG_HEADER * hdr,
-					       LOG_PAGE * log_pgptr);
+static int logpb_fetch_active_log_header_from_file (THREAD_ENTRY * thread_p, const char *db_fullname,
```

Comment:
A balance between name length and verbosity is needed. Having very long names makes the code difficult to skim-read and to indent.

E.g.:

logpb_fetch_header_from_file
logpb_fetch_header_from_page_server (or ps)
logpb_fetch_header_from_file_or_ps

Reply:
Shortened